tests/bootstrap.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Tests;
+
+use Dotenv\Dotenv;
+use LiveIntent\Services\Concerns\MocksRequests;
+
+require_once(dirname(__FILE__) . '/../vendor/autoload.php');
+
+class TestBootStrap
```

Comment:
I think i would prefer for the bootstrap to just be a script, I don't think we need a class here.

Reply:
just wanted to use `MockRequests` trait, to put a relevant `clearSnapshots` method, fine will write a plain script